tests/examples/awful/mouse/coords.lua

```diff
@@ -0,0 +1,13 @@
+screen[1]._resize {x = 175, width = 128, height = 96} --DOC_HIDE
+mouse.coords {x=175+60,y=60} --DOC_HIDE
+
+ -- Get the position
+print(mouse.coords().x)
+ 
```

Comment:
Actually, this is done on purpose. CMake drop empty lines and convert lines with trailing spaces into empty lines (not my idea, its how it behave).